test/acceptance/renumber.js

```diff
@@ -0,0 +1,12 @@
+describe('todo renumber', function() {
```

Comment:
Thanks @michaeljacobdavis 
> the domain objects

what is that? like storage is running and working?